### DISCUSSION TITLE
tx/rx freqs

### DISCUSSION THREAD

**Discussion by CGW6ACTUAL** [tarxvftech/FCS-152#72] [General]
Has anyone had any luck expanding the freqs a little beyond factory, maybe adding 1.25m band?

**Comment by tarxvftech** [comment]
No one has tried to my knowledge. Is there any reason to believe it's possible?

There have only been some abortive efforts to make a drop-in replacement radio module, and I was working on the software stack for UI and input fixes, but all efforts were dropped in favor of other projects.


(Why the sudden interest from two never-used-github-before users?)
